cpp/modmesh/toggle/pymod/wrap_profile.cpp

```diff
@@ -163,12 +163,43 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapTimeRegistry
 
 }; /* end class WrapTimeRegistry */
 
+class WrapCallProfiler : public WrapBase<WrapCallProfiler, CallProfiler>
```

Comment:
Expose CallProfiler to python so profiling util can be shared.
RadixTree is internal data structure, hence no need to expose this to python